litgpt/finetune/adapter.py

```diff
@@ -231,7 +231,8 @@ def fit(
         val_loss = validate(fabric, model, val_dataloader, dataclasses.replace(eval, max_iters=len(val_dataloader)))
         val_loss = f"{val_loss:.3f}"
     else:
-        validate(fabric, model, val_dataloader, dataclasses.replace(eval, max_iters=2))  # sanity check
+        print("Verifying settings ...")
```

Comment:
Shouldn't it also use `fabric.print` ?
If I am not mistaken a regular print, in case of multi-gpu setup, will repeat this print per each device.